quantum/eeconfig.h

```diff
@@ -51,9 +50,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define EECONFIG_LED_MATRIX_EXTENDED (uint16_t *)32
 #define EECONFIG_RGB_MATRIX_EXTENDED (uint16_t *)32
 
-// TODO: Combine these into a single word and single block of EEPROM
-#define EECONFIG_KEYMAP_UPPER_BYTE (uint8_t *)34
 // Size of EEPROM being used, other code can refer to this for available EEPROM
+// TODO: Reduce?
 #define EECONFIG_SIZE 35
```

Comment:
Should be fine to reduce
```suggestion
#define EECONFIG_SIZE 34
```